docs/en/observability/monitor-uptime-synthetics.asciidoc

```diff
@@ -5,9 +5,17 @@
 <titleabbrev>Synthetic monitoring</titleabbrev>
 ++++
 
-The {uptime-app} in {kib} lets you monitor user journeys and full page loads using our browser monitor,
-and also monitor the availability and response times of applications and services using our lightweight HTTP/S,
-TCP, and ICMP monitors.
+Synthetic monitoring extends traditional end-to-end testing techniques because it allows your tests to run continuously on the cloud.
+With Elastic's Synthetic Monitoring product, you can reuse the same journeys that you've used to validate the software on your machine
+to assert that your application continues to work after a deployment.
+
+You can use synthetic monitors to detect bugs caused by invalid states you couldn't anticipate and didn't write tests for.
+Synthetic monitors can also help you catch bugs in features that don't get enough traffic by allowing you to periodically simulate users' actions.
```

Comment:
Looks great without that extra sentence! You were right in removing it. Good call!